tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/core/postoffice/impl/WildcardAddressManagerUnitTest.java

```diff
@@ -464,6 +465,21 @@ public void setMessageLoadBalancingType(MessageLoadBalancingType messageLoadBala
 
       }
 
+      @Override
+      public Binding getBinding(String name) {
+         return null;
```

Comment:
even if its BindingsFake, seems weird not to implement this when the other methods are.

Reply:
that's not really needed, but ok.